bm/CMakeLists.txt

```diff
@@ -64,7 +64,7 @@ endfunction()
 
 c4_add_executable(ryml-bm-parse
     SOURCES bm_parse.cpp
-    LIBS ryml yaml yaml-cpp benchmark jsoncpp_lib nlohmann_json c4fs
+    LIBS ryml yaml yaml-cpp benchmark jsoncpp_static nlohmann_json c4fs
```

Comment:
Why did you need this? Never had the previous one fail. Maybe you configured with -DBUILD_SHARED_LIBS=OFF at the top level?

Reply:
I didn't configure with `-DBUILD_SHARED_LIBS=OFF` explicitly, but that was indeed the problem.  I'll revert this change.  But I can replicate this issue, steps:
```
git clone https://github.com/biojppm/rapidyaml.git
cd rapidyaml/
mkdir build
git submodule update --init --recursive
cd build
cmake -DRYML_DEV=ON -GNinja ..
grep BUILD_SHARED_LIBS CMakeCache.txt 
```

Outputs:
```
BUILD_SHARED_LIBS:BOOL=OFF
```

at the end.  Invoking the CMake command as:
```
cmake -DRYML_DEV=ON -DBUILD_SHARED_LIBS=ON -GNinja ..
```
does avoid this issue.

Reply:
I've removed this change.